tests/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.10)
 
 # Link runCSMCameraModelTests with what we want to test and the GTest and pthread library
 add_executable(runCSMCameraModelTests FramePluginTests.cpp FrameCameraTests.cpp)
-target_link_libraries(runCSMCameraModelTests usgscsm ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} pthread)
+if(WIN32)
+  option(CMAKE_USE_WIN32_THREADS_INIT "using WIN32 threads" ON)
+  option(gtest_disable_pthreads "Disable uses of pthreads in gtest." ON)
```

Comment:
I moved these options from the main CMakeLists.txt into here since they are gtest specific.

Reply:
I guess that this is also moot since the tests are turned off.